### ISSUE TITLE
Missing Brand Identity Document

### ISSUE THREAD

**Issue by davidegiovanni** [revas/ecosystem#17]
- [x] I am not here to ask for information or support.
- [x] I have searched in the issue track but found nothing.

## Benefits

A brand identity document helps understand the values of Revas as a project, its tone of voice and visual guidelines in order to develop coherent and cohesive contents.

@revas/designers

**Comment by samanthabaita** [comment]
@davidegiovanni Yes, it would be great to have a document with all the brand guidelines. :blush:
